Formula/singular.rb

```diff
@@ -0,0 +1,43 @@
+class Singular < Formula
+  desc "Computer algebra system for polynomial computations."
+  homepage "https://www.singular.uni-kl.de/"
+  url "http://www.mathematik.uni-kl.de/ftp/pub/Math/Singular/src/4-1-0/singular-4.1.0p3.tar.gz"
+  version "4.1.0"
+  sha256 "440164c850d5a1575fcbfe95ab884088d03c0449570d40f465611932ffd0bf80"
+  head "https://github.com/Singular/Sources.git"
+
+  option "without-test", "Skip build-time tests (not recommended)"
+  option "without-flint", "Build without flint support"
+
+  depends_on :xcode => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "gmp"
+  depends_on "ntl" => :recommended
```

Comment:
remove the `=> :recommended` please

Reply:
The authors recommend to compile with NTL (https://github.com/Singular/Sources/wiki/Building-Singular-from-source). Did I misunderstand the purpose of :recommended?

Reply:
Statistically speaking, the percentage of people who opt-out of `recommended` deps is very low, to the point it's not worth making deps `:recommended` initially until there's some evidence it's both merited & broadly desired. I presume that's what ILZ was swinging at.